tests/types/ember-concurrency-test.ts

```diff
@@ -3128,6 +3129,7 @@ module('integration tests', () => {
       debug = task(this, { debug: true }, async () => {});
       onState = task(this, { onState: () => {} }, async () => {});
       onStateNull = task(this, { onState: null }, async () => {});
+      waitFor = task({ waitFor }, async () => {});
```

Comment:
I think this is missing the `this` (otherwise, it's the same test case as the one further down):

```suggestion
      waitFor = task(this, { waitFor }, async () => {});
```